frontend/web/project/api.js

```diff
@@ -82,8 +82,47 @@ global.API = {
       const identify = new amplitude.Identify().set('email', id)
       amplitude.getInstance().identify(identify)
     }
-    flagsmith.identify(id)
-    flagsmith.setTrait('email', id)
+    API.flagsmithIdentify()
+  },
+  flagsmithIdentify() {
+    const user = AccountStore.model
+    if (!user) {
+      return
+    }
+
+    flagsmith
+      .identify(user.email, {
+        email: user.email,
+        organisations: user.organisations
+          ? user.organisations.map((o) => `"${o.id}"`).join(',')
+          : '',
+      })
+      .then(() => {
+        return flagsmith.setTrait(
+          'logins',
+          (flagsmith.getTrait('logins') || 0) + 1,
+        )
+      })
+      .then(() => {
+        const organisation = AccountStore.getOrganisation()
+        const emailDomain = `${user?.email}`?.split('@')[1] || ''
+        console.log(freeEmailDomains)
```

Comment:
We should probably remove this console log?

Reply:
Yep!